dataprovider/src/main/java/cm/aptoide/pt/dataprovider/ws/v7/promotions/GetPromotionAppsResponse.java

```diff
@@ -0,0 +1,87 @@
+package cm.aptoide.pt.dataprovider.ws.v7.promotions;
+
+import cm.aptoide.pt.dataprovider.model.v7.BaseV7Response;
+import cm.aptoide.pt.dataprovider.model.v7.listapp.App;
+import java.util.List;
+
+public class GetPromotionAppsResponse extends BaseV7Response {
+
+  private Datalist datalist;
```

Comment:
We already have a Datalist object for the response in the project. Can check BaseV7EndlessDataListResponse and examples that extend it.

Reply:
fixed